itl/command-plugins.conf

```diff
@@ -502,7 +502,8 @@ object CheckCommand "http" {
 		}
 		"-k" = {
 			value = "$http_header$"
-			description = "Any other tags to be sent in http header"
+			description = "Any other tags to be sent in http header (may be repeated)"
+			repeat_key = true
```

Comment:
Are you sure this is needed?

"If the argument value is an array, repeat the argument key, or not. Defaults to true (repeat)."
– https://icinga.com/docs/icinga-2/latest/doc/09-object-types/#checkcommand-arguments

Reply:
Argh ... probably not. But it is specified explicitly in several other places (even in the same config file). 

It also might have pretty strange results with the current default when someone repeats a key for a value where the plugin does _not_ allow repetition. But that's a different story.

Reply:
So... we'll probably close this one?

Reply:
The question is whether to close it or add the flag for clarification's sake. It doesn't hurt after all, and IMO it makes the usage of the command clearer - I wouldn't have brought this up at all had the repeat_key been specified for the command.

Reply:
Yes, it makes sense to clarify this in the descriptions. Please have a look how the other commands describe it.